### DISCUSSION TITLE
v0.5 to v0.6 migration guide?

### DISCUSSION THREAD

**Discussion by wyli** [Project-MONAI/MONAI#2575] [Q&A]
As breaking changes are introduced in v0.6, we should highlight them and provide a comprehensive guide for code migration.
This could be a wiki page summarizing the changes according to the tutorial (https://github.com/Project-MONAI/tutorials/pull/246).

(part of https://github.com/Project-MONAI/MONAI/issues/2402)

**Comment by wyli** [comment]
started a wiki page: https://github.com/Project-MONAI/MONAI/wiki/v0.5-to-v0.6-migration-guide

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Thanks for raising this topic. I think mainly 2 parts:
1. change the postprocessing transform to run with `decollate_batch` list and add `ToTensord` transform.
2. change the metrics computation methods to `__call__()`, `aggregate()` and `reset()`.

I will add code example to the wiki page.

Thanks.

**Comment by Nic-Ma** [comment]
Hi @wyli ,

I completed the wiki page with explanation and code examples.
Could you please help review it?
Please feel free to edit / add items.

Thanks in advance.

**Comment by wyli** [comment]
Thanks, I made some changes to the doc. I think we should regularly maintain/review this doc. I'm converting the ticket into a discussion.
